ls1intum/Artemis
# `Iris`: Exchange chat symbol with Iris logo

### Checklist
#### General


#### Client

### Motivation and Context



We want to raise awareness of IRIS.

### Description


As a first step, this PR replaces the generic chat bubble with a larger IRIS logo.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Exercise with IRIS enabled

1. Log in to Artemis
2. Navigate to the exercise
3. Verify that the IRIS logo is in the bottom right
4. Verify that it has a hover scale effect
5. Verify that it disappears if you click it, and the chat window pops up
6. Verify that it reappears if you close the window

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

Only TS9 works for IRIS.

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test



### Screenshots


![Arc 2023-11-23 at 17 07 38](https://github.com/ls1intum/Artemis/assets/23171488/61085c34-9891-4b3d-bad6-be74704a5860)
